### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by TenebrisN** [ArcPh1r3/RedGuyMod#24]
Really apreciate the new addings to character, just wanted to let you know that blood well healling still doesn't function in multiplayer for clients.
GL with new updates.

**Comment by InfiniteHouseplant** [comment]
Didnt even know mods were multiplayer compatible

**Comment by TenebrisN** [comment]
Good ones are compitable, the ones that made by lazyer ppl stay bugged forever. (of course it's not the case for this one bcs it's new)

**Comment by InfiniteHouseplant** [comment]
Does everyone have to have the same mod for it to work?

**Comment by TenebrisN** [comment]
yes, best to do it with r2modman and have your fiend or whoever you play with instal the same modpack, even configs have to be the same

**Comment by InfiniteHouseplant** [comment]
Thats kinda ridiculous but profiles exist so ig it aint too bad
